src/main/java/uk/gov/justice/digital/hmpps/whereabouts/repository/VideoLinkBookingEventRepository.kt

```diff
@@ -20,6 +22,27 @@ interface VideoLinkBookingEventRepository : JpaRepository<VideoLinkBookingEvent,
     ]
   )
   fun findByTimestampBetweenOrderByEventId(start: LocalDateTime, end: LocalDateTime): Stream<VideoLinkBookingEvent>
+
+  @Query(
+    nativeQuery = true,
+    value = """
+    select
+       *
+    from video_link_booking_event vlbe
+    left join video_link_booking_event later on
+      vlbe.video_link_booking_id = later.video_link_booking_id
+      and vlbe.timestamp < later.timestamp
+    where
+      later.video_link_booking_id is null
+      and vlbe.main_start_time between :start and :end
+      and vlbe.event_type != 'DELETE'
+    order by vlbe.main_start_time asc
+    """
+  )
```

Comment:
bit gnarly using raw sql rather than jpql but it was making my head hurt and this should be relatively short lived - once we move to storing all booking data in BVL